feature-configs/5g-ref-nw/profile-base/du-small/sriov/sriov-nw-du-fec-mh-u.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-nw-du-fec-mh-u
+  namespace: openshift-sriov-network-operator
```

Comment:
Need to fix the operator.

Reply:
Removed all the fec (far edge cloud) from the names to avoid confusion with forward error correction